src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -296,6 +298,9 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
     }
 
     void nn_planar() {
+        const bool is_i8 = jcp_.src_prc == InferenceEngine::Precision::I8
+                           && jcp_.dst_prc == InferenceEngine::Precision::I8;
+
```

Comment:
I believe we can support u8 as well with current implementation. Just relax conditions.

Reply:
Done